src/main/java/seedu/address/model/ContactList.java

```diff
@@ -28,6 +35,13 @@ public class ContactList implements ReadOnlyContactList {
         courseMates = new UniqueCourseMateList();
     }
 
+    /*
+     * Temporary measure
+     * Could possibly move to something like GroupList
+     */
+    private final UniqueGroupList groups = new UniqueGroupList();
```

Comment:
Yes, it's better to create a new class called GroupList. Recommended to change during v1.2 before it's too late.

Reply:
ok thanks. Yeah I just wanted to make sure the GUI works first so I hardcoded some stuff